### ISSUE TITLE
"The results are comparable with the popular MOTChallenge benchmarks."

### ISSUE THREAD

**Issue by mattans** [cheind/py-motmetrics#92]
I am not sure this sentence is correct.

In this package, MOTP is implemented as appears in the original paper:

https://cvhci.anthropomatik.kit.edu/~stiefel/papers/ECCV2006WorkshopCameraReady.pdf
http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.367.6279&rep=rep1&type=pdf

This "version" of MOTP has a misleading name, it is not precision but misalignment. This MOTP is meant to be minimized, as it has object-hypothesis **distances** in the numerator.

However, in MOT Challenge, what they call MOTP is actually (1 - MOTP), so they try to **maximize** its value. This was confirmed with them by mail that 100% MOTP actually corresponds to 0 misalignment.

In conclusion, I believe you are aligned with the paper, but not with MOT Challenge. I observed this when I saw that superior results have lesser MOTP, while in MOT Challenge they note that higher MOTP is better.

**Comment by jvlmdr** [comment]
This is documented in the project readme:
https://github.com/cheind/py-motmetrics#motchallenge-compatibility

> Metric MOTP seems to be off. To convert compute (1 - MOTP) * 100. MOTChallenge benchmarks compute MOTP as percentage, while py-motmetrics sticks to the original definition of average distance over number of assigned objects [1].

**Comment by mattans** [comment]
Thanks.
I didn't keep reading after "The results are comparable with the popular MOTChallenge benchmarks" and "The metrics have been aligned with what is reported by MOTChallenge benchmarks".
Maybe consider adding an asterisk to these statements...
